### PR TITLE
Collapsible recent pages

### PR DESCRIPTION
**Pull request by mejo-**
### 📝 Summary

* Members widget moved to the top and remove its caption
* Decreased top margin for captions

Fixes: #835

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [x] Tests (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [x] Documentation is not required

### CODE REVIEW

File: src/components/Page/LandingPageWidgets/RecentPagesWidget.vue [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 	<div class="recent-pages-widget">
-		<WidgetHeading :title="t('collectives', 'Recent pages')" :first="true" />
-		<div class="recent-pages-widget-container">
+		<a class="recent-pages-title"
+			@keydown.enter="toggleWidget"
+			@click="toggleWidget">
+			<WidgetHeading :title="t('collectives', 'Recent pages')" />
+			<div class="toggle-icon">
+				<ChevronDownButton :size="24"
+					:title="t('collectives', 'Expand recent pages')"
```
Comment (max-nextcloud) [inline src/components/Page/LandingPageWidgets/RecentPagesWidget.vue:9]: Maybe adjust the title to the toggle state?

```suggestion
					:title="showRecentPages ? t('collectives', 'Collapse recent pages') : t('collectives', 'Expand recent pages')"
```

**Review by max-nextcloud — APPROVED** [review]
Looks good. Nice and cleanly separated into the differen commits. I just have one small comment.

File: src/components/Page/LandingPageWidgets/RecentPagesWidget.vue (hunk 1 above)
Comment (mejo-) [reply src/components/Page/LandingPageWidgets/RecentPagesWidget.vue:9]: Nice idea! I went with setting the `aria-label` for the `a` tag according to your suggestion instead - it's the better place for a label anyway, given that it's the actionable element.

**Comment by punkyard** [comment]
please, remove Recent pages, Landing page and contributors from the edition frame of Collectives.

If people need this, could it be displayed for them in the left frame of the screen, with the menu?

**Pull request merged**